patches/chromium/fix_desktop_capturer_show_own_window.patch

```diff
@@ -0,0 +1,28 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: George Xu <george.xu@slack-corp.com>
+Date: Wed, 11 Dec 2024 15:00:09 -0800
+Subject: fix: desktopCapturer.getSources does not return electron window on
+ Windows
+
+This patch is to fix issue https://github.com/electron/electron/issues/36037, where `desktopCapturer.getSources` does not return the electron window on Windows.
+
+This patch likely cannot be upstreamed. It can be removed if we can expose the ShouldEnumerateCurrentProcessWindows method in Electron's desktopCapturer API, or if desktopCapturer can use an alternative source API for getSources on Windows.
+
+partially reverts https://chromium-review.googlesource.com/c/chromium/src/+/2907415
```

Comment:
Sorry I didn't pay attention to the CL before, IIUC the behavior should remain unchanged if we call `NativeDesktopMediaList` with `add_current_process_windows` set to `true` or is that not the case, in the `false` case it would rely on the capturer to get the current process windows and that needs this patch. Would like to confirm if we can avoid this patch by changing the callsite [here](https://github.com/electron/electron/blob/69479b2fb7c5bbbf7e9bfff51f6867829439059c/shell/browser/api/electron_api_desktop_capturer.cc#L288-L289) ?